interface/main/messages/messages.php

```diff
@@ -23,614 +24,686 @@
  * @author OpenEMR Support LLC
  * @author Roberto Vasquez <robertogagliotta@gmail.com>
  * @author Rod Roark <rod@sunsetsystems.com>
- * @author Brady Miller <brady.g.miller@gmail.com>
+ * @author Brady Miller <brady@sparmy.com>
+ * @author Ray Magauran <magauran@medfetch.com>
  * @link http://www.open-emr.org
  */
```

Comment:
Lets shorten this license/copyright/author header to just:
/**
 * Message and Reminder Center UI
 *
 * 2013/02/08 Minor tweaks by EMR Direct to allow integration with Direct messaging
 * 2013-03-27 by sunsetsystems: Fixed some weirdness with assigning a message recipient,
 *   and allowing a message to be closed with a new note appended and no recipient.
 *
 * @package OpenEMR
 * @link    http://www.open-emr.org
 * @author  OpenEMR Support LLC
 * @author  Roberto Vasquez <robertogagliotta@gmail.com>
 * @author  Rod Roark <rod@sunsetsystems.com>
 * @author  Brady Miller <brady.g.miller@gmail.com>
 * @author  Ray Magauran <magauran@medfetch.com>
 * @copyright Copyright (c) 2010 OpenEMR Support LLC
 * @copyright Copyright (c) 2017 MedEXBank.com
 * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
 */